Revit_Core_Engine/Convert/Revit/FromRevit/DraftingInstance.cs

```diff
@@ -65,7 +65,7 @@ public static DraftingInstance DraftingInstanceFromRevit(this FilledRegion fille
                 curves.Add(loop.FromRevit());
             }
 
-            List<PlanarSurface> surfaces = BH.Engine.Geometry.Create.PlanarSurface(curves);
+            List<PlanarSurface> surfaces = BH.Engine.Geometry.Create.PlanarSurface(curves, settings.DistanceTolerance);
```

Comment:
Before line 69,  could you please add a check if (surfaces.Count == 0) and if so, then raise a warning and set the instance's location to `null`? It seems to make more sense than returning the location equal to an empty `PolySurface`. Or am I confusing things?